{
    "Modified": "2014-12-02T21:59:49.497-05:00",
    "Published": "2013-10-09T10:53:20.230-04:00",
    "access": {
        "authentication": "NONE",
        "complexity": "MEDIUM",
        "vector": "NETWORK"
    },
    "capec": [],
    "cvss": "4.3",
    "cvss-time": "2013-10-09T11:23:48.563-04:00",
    "cwe": "CWE-399",
    "id": "CVE-2013-2099",
    "impact": {
        "availability": "PARTIAL",
        "confidentiality": "NONE",
        "integrity": "NONE"
    },
    "last-modified": "2016-06-08T21:59:01.407-04:00",
    "map_cve_nessus": {
        "nessus_script_family": "Fedora Local Security Checks",
        "nessus_script_file": "fedora_2014-16390.nasl",
        "nessus_script_id": "80061",
        "nessus_script_name": "Fedora 20 : python-tornado-2.2.1-7.fc20 (2014-16390)"
    },
    "map_cve_openvas": {
        "openvas_script_family": "Fedora Local Security Checks",
        "openvas_script_file": "gb_fedora_2013_21415_python3_fc18.nasl",
        "openvas_script_id": "867086",
        "openvas_script_name": "Fedora Update for python3 FEDORA-2013-21415"
    },
    "map_cve_oval": {
        "cpeid": "",
        "ovalclass": "patch",
        "ovalid": "oval:org.mitre.oval:def:19404",
        "ovaltitle": "USN-1985-1 -- python3.3 vulnerabilities"
    },
    "map_cve_redhat": {
        "redhatid": "RHSA-2014:1690",
        "redhatovalid": "",
        "redhatupdatedesc": ""
    },
    "map_cve_scip": {
        "scipid": "65232",
        "sciplink": "http://www.scip.ch/en/?vuldb.65232"
    },
    "map_cve_ubuntu": {
        "ubuntuid": "USN-1985-1"
    },
    "ranking": [
        [
            {
                "circl": 2
            }
        ]
    ],
    "references": [
        "http://bugs.python.org/issue17980",
        "https://bugzilla.redhat.com/show_bug.cgi?id=963260",
        "https://access.redhat.com/errata/RHSA-2016:1166",
        "http://www.ubuntu.com/usn/USN-1985-1",
        "http://www.ubuntu.com/usn/USN-1984-1",
        "http://www.ubuntu.com/usn/USN-1983-1",
        "http://www.openwall.com/lists/oss-security/2013/05/16/6",
        "http://rhn.redhat.com/errata/RHSA-2014-1690.html"
    ],
    "summary": "Algorithmic complexity vulnerability in the ssl.match_hostname function in Python 3.2.x, 3.3.x, and earlier, and unspecified versions of python-backports-ssl_match_hostname as used for older Python versions, allows remote attackers to cause a denial of service (CPU consumption) via multiple wildcard characters in the common name in a certificate.",
    "vulnerable_configuration": [
        {
            "id": "cpe:2.3:a:python:python:3.3.0",
            "title": "Python 3.3.0"
        },
        {
            "id": "cpe:2.3:a:python:python:3.3.1",
            "title": "Python 3.3.1"
        },
        {
            "id": "cpe:2.3:a:python:python:3.3.2",
            "title": "Python 3.3.2"
        },
        {
            "id": "cpe:2.3:a:python:python:3.2.0",
            "title": "Python 3.2.0"
        },
        {
            "id": "cpe:2.3:a:python:python:3.2.1",
            "title": "Python 3.2.1"
        },
        {
            "id": "cpe:2.3:a:python:python:3.2.2",
            "title": "Python 3.2.2"
        },
        {
            "id": "cpe:2.3:a:python:python:3.2.4",
            "title": "Python 3.2.4"
        },
        {
            "id": "cpe:2.3:a:python:python:3.2.5",
            "title": "Python 3.2.5"
        },
        {
            "id": "cpe:2.3:a:python:python:3.2.3",
            "title": "Python 3.2.3"
        },
        {
            "id": "cpe:2.3:o:canonical:ubuntu_linux:13.04",
            "title": "Canonical Ubuntu Linux 13.04"
        },
        {
            "id": "cpe:2.3:o:canonical:ubuntu_linux:12.04:-:lts",
            "title": "Canonical Ubuntu Linux 12.04 LTS (Long-Term Support)"
        },
        {
            "id": "cpe:2.3:o:canonical:ubuntu_linux:12.10",
            "title": "Canonical Ubuntu Linux 12.10"
        }
    ],
    "vulnerable_configuration_cpe_2_2": [
        "cpe:/a:python:python:3.3.0",
        "cpe:/a:python:python:3.3.1",
        "cpe:/a:python:python:3.3.2",
        "cpe:/a:python:python:3.2.0",
        "cpe:/a:python:python:3.2.1",
        "cpe:/a:python:python:3.2.2",
        "cpe:/a:python:python:3.2.4",
        "cpe:/a:python:python:3.2.5",
        "cpe:/a:python:python:3.2.3",
        "cpe:/o:canonical:ubuntu_linux:13.04",
        "cpe:/o:canonical:ubuntu_linux:12.04:-:lts",
        "cpe:/o:canonical:ubuntu_linux:12.10"
    ]
}